test/focus-trap-react.test.js

```diff
@@ -317,6 +323,156 @@ describe('FocusTrap', () => {
         expect(document.body).toHaveFocus();
       });
     });
+
+    it('Does not call onPostActivate() until checkCanFocusTrap() has completed', async () => {
+      let hasActivated = false;
+      let hasRunPostActivate = false;
+      render(
+        <FocusTrapExample
+          focusTrapOptions={{
+            checkCanFocusTrap: () => pause(500),
```

Comment:
Is there a reason it has to be this long? Wouldn't just 5 or 10ms suffice?

Reply:
5ms is probably OK...

Reply:
That would good. I'd like to avoid introducing unnecessary test time duration as much as possible. So let's use the shortest delays possible.